tests/integration/static_ip_address_test.py

```diff
@@ -16,13 +16,32 @@
 #
 
 import copy
+import pytest
 
 from libnmstate import netapplier
 
 from .testlib import assertlib
 from .testlib import statelib
 from .testlib.statelib import INTERFACES
 
+@pytest.fixture
+def setup_eth1_ipv4():
+    desired_state = {
+        INTERFACES: [
+            {
+                'name': 'eth1',
+                'type': 'ethernet',
+                'state': 'up',
+                'ipv4': {
+                    'enabled': True,
+                    'address': [
+                        {'ip': '192.168.122.251', 'prefix-length': 24}
```

Comment:
Please use a designated test address, such as 192.0.2.251, see 
https://tools.ietf.org/html/rfc5737#section-3 - this will reduce problems with address conflicts.